internal/spokes/spokes.go

```diff
@@ -810,6 +810,11 @@ func (r *spokesReceivePack) readPack(ctx context.Context, commands []command, ca
 
 	if r.isFsckConfigEnabled() {
 		args = append(args, "--strict")
+		strictArgs := r.config.GetPrefix("receive.fsck")
```

Comment:
The argument expected is `strict="missingEmail=ignore,badTagName=error"` so you will need to pass an `=` as an extra argument.

What do you think about something like:

```suggestion
strictArgs := r.config.GetPrefix("receive.fsck")
if len(strictArgs) > 0 {
args= append(args,"=", strings.Join(strictArgs, ","))
}
```

Reply:
This way, you will end up with a command like:

```
--strict="missingEmail=ignore,badTagName=error"
```

that I think it's the expected syntax

Reply:
Yes exactly, great catch.